nuttx/arch/arm/src/stm32/chip/stm32l4x6xx_memorymap.h

```diff
@@ -123,11 +123,11 @@
 /* APB2 Base Addresses **************************************************************/
 
 #define STM32_SYSCFG_BASE    0x40010000     /* 0x40010000-0x4001002f SYSCFG */
-#define STM32_VREF_BASE      0x40010030     /* 0x40010030-0x400101ff VREF */
+#define STM32_VREF_BASE      0x40010030     /* 0x40010030-0x400101ff VREFBUF */
 #define STM32_COMP_BASE      0x40010200     /* 0x40010200-0x400103ff COMP */
 #define STM32_EXTI_BASE      0x40010400     /* 0x40010400-0x400107ff EXTI */
 #define STM32_FIREWALL_BASE  0x40011c00     /* 0x40011c00-0x40011fff FIREWALL */
-#define STM32_SDMMC_BASE     0x40012800     /* 0x40012800-0x40012bff SDMMC */
+#define STM32_SDMMC_BASE     0x40012800     /* 0x40012800-0x40012bff SDMMC1 */
```

Comment:
SMMC_BASE would be the same address as SDMMC1 but is not SDMMC1.  The original was correct.  I would recommend that you leave both files with only comment changes unchanged.